exonum/src/events/network.rs

```diff
@@ -281,16 +206,128 @@ impl NetworkPart {
                     }
                 }
             });
+        RequestHandler { handler: tobox(requests_handler) }
+    }
+
+    fn connect_to_peer(
```

Comment:
In my opinion this function is looking more appropriate as a part of the `ConnectionPool`.